client/app/cloud/project/add/cloud-project-add.less

```diff
@@ -44,6 +44,11 @@
       left: 0;
       bottom: 0;
     }
+
+    .oui-button {
```

Comment:
According to the UI-Kit, these colors aren't allowed.

Is there any reason to still apply theses values?

ref: https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/ovh-ui-kit/button

Reply:
Well the page looks very different from the usual pages in manager. And i added these colours to retain the ui it had previously. you can see this in the ticket MBE-205. 
Will have to get a go ahead from CX team and new task if i have to change the whole page and bring it to ui-kit standards.

Reply:
I agree but we definitely have to avoid this type of addition. At a certain point theses rules should be removed. I know that you've been asked to turn these elements into a `<oui-button>` but in my opinion we should handle it when the whole page will be reviewed.